package.json

```
@@ -29,15 +29,16 @@
         "@emotion/core": "^10.0.28",
         "@guardian/atoms-rendering": "^1.3.3",
         "@guardian/automat-client": "^0.2.16",
-        "@guardian/consent-management-platform": "^3.0.4",
+        "@guardian/consent-management-platform": "^3.4.3",
         "@guardian/discussion-rendering": "^2.2.23",
         "@guardian/shimport": "^1.0.2",
-        "@guardian/src-button": "^0.17.0",
-        "@guardian/src-ed-lines": "^0.18.0-rc.0",
-        "@guardian/src-foundations": "^0.17.0",
-        "@guardian/src-link": "^0.18.0-rc.0",
-        "@guardian/src-svgs": "^0.17.0",
-        "@guardian/src-text-input": "^0.18.0-rc.0",
+        "@guardian/src-button": "^1.5.0",
+        "@guardian/src-ed-lines": "^1.5.0",
+        "@guardian/src-foundations": "^1.5.0",
+        "@guardian/src-link": "^1.5.0",
+        "@guardian/src-icons": "^1.5.0",
+        "@guardian/src-svgs": "^1.5.0",
```

you could alias this, so:

```suggestion
        "@guardian/src-svgs": "npm:@guardian/src-icons@^1.5.0",
```